commands/login/index.js

```diff
@@ -37,8 +37,10 @@ program
     } catch (e) {
       return logError(e);
     }
+    const conf = {};
+    conf[process.env.LOSANT_API_URL]= { apiToken: api.getOption('accessToken') };
```

Comment:
seems weird, shouldn't you be pulling this from the api client? and not assuming LOSANT_API_URL?